db/model.js

```diff
@@ -16,4 +16,22 @@ const fetchReviews = () => {
     })
 }
 
-module.exports = { fetchCategories, fetchReviews }
\ No newline at end of file
+const fetchReviewById = (id) => {
+    const parsedId = parseInt(id)
+    
+    if (id > 24) {
+        return Promise.reject({status: 404, msg: "invalid review ID"})
+    };
```

Comment:
This works for now but what if later we are able to add reviews to the database this id will go higher than 24 and therefore trigger this error - is there another way we can check if the id exists? If it doesn't exist, we won't receive anything back in our rows array.